src/switch/switch.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let inputClass = null;
+  export let trackClass = null;
+  export let thumbClass = null;
+
+  export let value = false;
+  export let disabled = false;
+  export let slotLeft = false;
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<label class={classes('switch', _class)}>
+  {#if slotLeft}
+    <slot />
+  {/if}
+  <input
+    bind:checked={value}
+    type="checkbox"
+    class={inputClass}
+    {disabled}
+    on:change={(e) => dispatch('change', { value: e.target.checked })}
```

Comment:
Same comment as in #14 about dispatched events
```suggestion
    on:change
```

Reply:
Same thing, to be fixed later